src/base/utils/fs.h

```diff
@@ -32,8 +32,14 @@
  * Utility functions related to file system.
  */
 
+#include <libtorrent/fwd.hpp>
+
 #include <QString>
 
+#include "base/3rdparty/expected.hpp"
```

Comment:
Maybe provide some wrapper header `base/expected.h` and add `using namespace nonstd;` there?

Reply:
I don't see the point of doing so. It would be easier in current form to migrate to the real `std::expected`.

Reply:
Using current form you have to replace all `nonstd` with `std`. Using proposed wrapper you have to change only aliasing inside `base/expected.h`:
```c++
#include "base/3rdparty/expected.hpp"
using expected = nonstd::expected;
// can be changed to
#include <expected>
using expected = std::expected;
```

Reply:
However, I do not insist on this.

Reply:
We use `std::optional` instead of just `optional` right? I suppose we should follow that with `std::expected`.

Reply:
Apparently, this was done thoughtlessly... I would be satisfied with just `optional`.
However, we can make a decision later, and leave it as it is now.